src/components/Form/Form.js

```diff
@@ -13,16 +13,49 @@ const form = () => {
   const { quizForm, step } = formState;
 
   const nextStep = () => {
+    const currStep = step + 1;
     updateFormState({
       ...formState,
-      step: step + 1
+      step: currStep
     });
+
+    initDefaultCheckedInput(quizForm[formState.step]['values'], step, currStep);
   };
 
   const prevStep = () => {
+    const currStep = step - 1;
     updateFormState({
       ...formState,
-      step: step - 1
+      step: currStep
+    });
+
+    initDefaultCheckedInput(quizForm[formState.step]['values'], step, currStep);
+  };
+
+  const initDefaultCheckedInput = (currentFormPage, step, currStep) => {
+    let updateFormPage = { ...currentFormPage };
+    const updatedForm = {
+      ...formState
+    };
+
+    for (let key in updateFormPage) {
```

Comment:
errrm ... should not `for..in` in general imho

Reply:
sure, I am trying to keep it simple, which means I have only Parent Business Login and trying to propagate the methods to the Functional Component

Reply:
`updateFormPage` is object not array

Reply:
I think `Object.entries` would be your better choice here

Reply:
I can replace `for ... in ` with Object.entries()